patches/aports.config

```diff
@@ -1,3 +1,4 @@
+9.1.2,364ac320139da4a9dfd68ea399267bd5056a670e
```

Comment:
would need to change that to `9.2.0` and use compatible commit sha aports from https://github.com/alpinelinux/aports/tree/master/community/qemu

In this case: https://github.com/alpinelinux/aports/commit/334b162677f79271980f61d410dc136b98b93fc4

Reply:
This time, also made sure the `buildkit` variant builds locally, so should be all good :)